packages/waku/src/router/define-router.ts

```diff
@@ -205,13 +206,23 @@ export function unstable_defineRouter(fns: {
     };
   }[];
   let cachedPathConfig: MyPathConfig | undefined;
+  const buildOptions = unstable_getBuildOptions();
```

Comment:
I think we should add this in one of e2e fixtures. Can you try? Let me know if you need help.

Reply:
This is used in `examples/07_cloudlfare`

https://github.com/wakujs/waku/blob/174c065b4da955def6261fd92e8d9a6807eca01d/examples/07_cloudflare/src/lib/waku.ts#L5-L7

I'm not sure the intended external usage though. Is this to detect ssg phase from runtime?

Reply:
I think it's to change the behavior for build or for runtime. It's not very nice, but sometimes it can be a workaround.

The `unstable_getBuildOptions` API is very unstable and it's very likely that we will change the API, but currently, it's only the way.